### ISSUE TITLE
Need to see the full upcoming list

### ISSUE THREAD

**Issue by chriscaddell** [JasonZigelbaum/jqbx-issues#17]
When you are in a room, it's important to be able to see what is upcoming (from everyone, not just yourself).

**Comment by JasonZigelbaum** [comment]
Would something like the following accomplish what you're thinking:

1. Hover over a user image in the "DJ Queue" in the center of the screen.
2. After 1 second or so a modal shows up showing the track that they have "On Deck" (it can change but it is = the track at the top of his/her queue when you hovered over the image).

If that syncs up with that you're thinking then great. If you had something else in mind I'm curious to hear more about it... let me know what you think either way.

Thanks!

**Comment by jtrutwin-zz** [comment]
You want to be able to see what people have queued up?  I don't think that's a feature I'd like.  Only for mods maybe.

**Comment by chriscaddell** [comment]
Yes, that would be really interesting. Also, a tab could be added where the
"history" tab is. This tab would list would is upcoming based on who has
said they would like to be DJ. Per Josh's comment below, that would be
handy for mods as well to see what was upcoming.
